examples/vmseries_standalone/outputs.tf

```diff
@@ -1,4 +1,9 @@
 output "vmseries_public_ips" {
   description = "Map of public IPs created within `vmseries` module instances."
   value       = { for k, v in module.vmseries : k => v.public_ips }
+}
+
+output "vm_series_ipv6_addresses" {
```

Comment:
```suggestion
output "vmseries_ipv6_addresses" {
```

Reply:
I suggested change of the name to have the same standard as we have for `vmseries_public_ips`